wolox-books-vue/src/services/booksService.js

```diff
@@ -0,0 +1,17 @@
+import { create } from 'apisauce'
+import { LocalStorageService } from './localStorageService'
+
+export class BooksService {
+  localStorageService = new LocalStorageService()
+  token = this.localStorageService.getValue(this.localStorageService.SESSION_TOKEN)
+
+  api = create(
+    {
+      baseURL: 'https://wbooks-api-stage.herokuapp.com/api/v1/books',
```

Comment:
you must have one class (or constant) for a global api and use it for the different resources of your app. But use only one instance of it